subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/AbstractIntegrationSpec.groovy

```diff
@@ -392,13 +392,27 @@ class AbstractIntegrationSpec extends Specification {
         result.assertHasPostBuildOutput(string.trim())
     }
 
-    void useRepositoryMirrors() {
+    void useRepositoryMirrors(boolean global = false) {
+        if (System.getenv("CI") == null) {
```

Comment:
I think we should always add the init script, even if it won't have any effect on developer machines. If we don't, then it will be really hard to reproduce some errors locally.

Reply:
Done.